taxcalc/reforms/Trump2017.json

```diff
@@ -0,0 +1,75 @@
+// Title: 2017 Trump Administration Tax Reform Proposal (potential)
+// Reform_File_Author: Cody Kallen
+// Reform_Reference: 1-page hand-out from the White House briefing
+// Reform_Description:
+// -  New personal income tax schedule (regular/non-AMT/non-pass-through) (1) 
+// -  New pass-through income tax schedule (2) 
+// -  Repeal Alternative Minimum Tax (3) 
+// -  Repeal Net Investment Income Tax (4) 
+// -  Double the Standard Deduction (5)
+// -  Eliminate all itemized deductions except mortgage interest and charitable contributions (6)
+// Reform_Parameter_Map:
+// - 1: _II_rt*
+// - 2: _PT_*
+// - 3: _AMT_*
+// - 4: _NIIT_rt
+// - 5: _STD
+// - 6: _ID_*
+{
+  "policy": {
+        "_II_rt1":
+            {"2017": [0.10]},
+        "_II_rt2":
+            {"2017": [0.10]},
+        "_II_rt3":
+            {"2017":  [0.25]},
+        "_II_rt4":
+            {"2017":  [0.25]},
+        "_II_rt5":
+            {"2017": [0.25]},
+        "_II_rt6":
+            {"2017": [0.35]},
+        "_II_rt7":
+            {"2017": [0.35]},
+        "_PT_rt1":
+            {"2017": [0.15]},
```

Comment:
I think _PT_rt1 and _PT_rt2 should probably be 0.1. I doubt they want to tax pass through income at a higher rate than wage and salary income